### ISSUE TITLE
training labels for accent, gender ..etc

### ISSUE THREAD

**Issue by mohammed-bahumaish** [voicepowered-ai/VibeVoice-finetuning#9]
Thanks for the great work! 

is it possible to use training labels for different accents/dialects/gender/age while training/fine-tuning the model and then reproduce that accent during inference?

**Comment by jpgallegoar-vpai** [comment]
I have not tested this myself and am unaware of anyone else trying it, although it is an idea I had in mind. I'm very confident the model would generalize to any given tags or custom system prompts, due to that it's already extremely general in nature. Please share your results if you decide to go through with it. One option is to change the system prompt of the training pairs, another one is to add tags inside the prompt (next to the speaker for example: "Speaker 1 (angry): ...", and another option is to add the tags inside the text itself like "Speaker 1: (angry) HEY! (sad) bla bla..."
